README.md

```diff
@@ -7,15 +7,75 @@ The Microsoft Linux Package Repositories are hosted on PMC ([packages.microsoft.
 ## Configuring the repository on your Linux system 
 
 See how to [host/install/upgrade](https://docs.microsoft.com/en-us/windows-server/administration/linux-package-repository-for-microsoft-software) Microsoft's Linux software using your distribution's standard package management tools.  
+In short you may enable Microsoft's Production repository for your distribution / version by installing the `packages-microsoft-prod.[rpm|deb]` package found at the appropriate [/config/](https://packages.microsoft.com/config/) subdirectory, and there may be additional / alternate repositories you can enable by making the `.repo|.list` files available to your package manager.
 
 Microsoft's Linux Software Repository is comprised of multiple repositories: 
 
-**prods** – These Production repositories (e.g. Ubuntu, Fedora, RHEL, etc.) are designated for packages intended to be used in production. These packages are commercially supported by Microsoft under the terms of the applicable support agreement or program that you have with Microsoft. The prod repositories can be located via hierarchical folder structure (e.g. https://packages.microsoft.com/fedora/36/prod/).
+* **prod** – These Production repositories (e.g. Ubuntu, Fedora, RHEL, etc.) are designated for packages intended to be used in production.
+  These packages are commercially supported by Microsoft under the terms of the applicable support agreement or program that you have with Microsoft.
+  The prod repositories can be located via hierarchical folder structure (e.g. https://packages.microsoft.com/fedora/36/prod/).
 
-**mssql-server** – These repositories contain packages for Microsoft SQL Server on Linux - See also: [SQL Server on Linux.](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-overview) 
+* **insiders-fast/insiders-slow** – These repositories provide a way to preview upcoming features for software released into the Production repos.
+  Packages generally flow from `insiders-fast` -> `insiders-slow` -> `prod`, but note that some software in the prod repos may not use these repos, and not all versions released here will be promoted to the next stage.
+  _NOTE: Not intended for production use._
```

Comment:
I prefer to use `_` for italics and `**` for bold because that's how asciiDoc has trained me to think. That seems to be the more almost-cross-compatible convention; for example the Markdown variant that Teams uses also uses a single asterisk for bold. And if you use the GitHub formatting buttons it translates into `_` and `**`.

I don't think _I_ ever used a single `*` for italics here, but I see there is one in this doc already. I would prefer to switch it to `_` for consistancy.